src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.openmodloader.loader.launch;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OMLDevClientLaunchHandler extends OMLClientLaunchHandler {
+
+	@Override
+	public Path[] identifyTransformationTargets() {
+		Path[] basePaths = super.identifyTransformationTargets();
+		List<Path> paths = new ArrayList<>(Arrays.asList(basePaths));
```

Comment:
Very tiny thing, does this need to be wrapped in an ArrayList?

Reply:
Nah, just a way to combine the elements. If you can think of a better way go for it

Reply:
Oh, I mean could it not just be `Arrays.asList(basePaths)`?

Reply:
I don't believe you can add elements to those lists... They're a special wrapper